### ISSUE TITLE
[1.19.2 Forge] Music Triggers audio outputs to default regardless of chosen output

### ISSUE THREAD

**Issue by KxttyKxt** [TheComputerizer/Music-Triggers#159]
**Describe the bug**
This problem is twofold. First, let's assume the selected audio output channel by the game is Default, and that Default is my PC speakers.

If I navigate to the **pause menu -> Options -> Music & Sounds** and click the `Device: ...` option to change it, the Minecraft sounds update accordingly, but the Music Triggers sounds will continue to output to the default channel, my speakers.

If from game launch, another option is already specifically chosen (for example, `Headphones (Realtek(R) Audio)`) instead of Default, Music Triggers still sends the music to the default audio output channel, my speakers.

**To Reproduce**
1. Produce a situation in which Music Triggers controls the currently playing Music (ex. Menu Screen, certain biome, etc.)
2. Go to **Options -> Music & Sounds** and click `Device: ...` to change the game's audio output channel.
3. Witness bug.

**Additional context**
Minecraft Version: 1.19.2
Forge Version: 43.3.5
Optimization mods such as Embeddium are present in the modpack, but I do not know if this would have any bearing on the exhibited behavior by Music Triggers.

**Comment by TheComputerizer** [comment]
Yeah that's a known issue and its due to how the mod writes the audio output. It's not unfixable, but it will require some overhaul that I was going to try for 6.4

**Comment by TheComputerizer** [comment]
This is still an issue in 7.0.0 but will be fixed sometime in the next few weeks
